example/build.gradle

```diff
@@ -2,7 +2,7 @@ apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
 
 android {
-    compileSdkVersion 33
+    compileSdk 34
```

Comment:
webkit requires minimum compileSdk version of 34. Also, `compileSdkVersion` is deprecated in favor of `compileSdk`.